index.src.html

```diff
@@ -749,6 +755,166 @@ <h3 id="try-delivery" algorithm>
   8.  Return "`Failure`".
 </section>
 
+<section>
+  <h2 id="observers">Reporting Observers</h2>
+
+  A <dfn>reporting observer</dfn> observes some types of <a>reports</a> from
+  JavaScript, and is represented in JavaScript by the {{ReportingObserver}}
+  object.
+
+  Each <a>environment settings object</a> has a <dfn>registered reporting
```

Comment:
For PO we use "Each ECMAScript global environment has": https://w3c.github.io/performance-timeline/#performance-timeline

Reply:
Okay, I'll use "ECMAScript global environment" instead.